pkg/resources/account.go

```diff
@@ -335,42 +357,37 @@ func ReadAccount(d *schema.ResourceData, meta interface{}) error {
 
 // UpdateAccount implements schema.UpdateFunc.
 func UpdateAccount(d *schema.ResourceData, meta interface{}) error {
-	db := meta.(*sql.DB)
-	client := sdk.NewClientFromDB(db)
-	ctx := context.Background()
-
-	id := helpers.DecodeSnowflakeID(d.Id()).(sdk.AccountObjectIdentifier)
+	/*
```

Comment:
do we have to leave this as a comment? if so, please add a description at the top of the comment about why we are leaving this

Reply:
made a note